### ISSUE TITLE
Must not be accessed before initialization

### ISSUE THREAD

**Issue by napruzzese** [AkrilliA/laravel-beyond#100]
When a person try to create Model, Enum etc for a Domain that not works!

```
php artisan beyond:make:enum Users/UserStatusEnum

 ┌ On which domain do you want to add your Enum ────────────────┐
 │ Users                                                        │
 └──────────────────────────────────────────────────────────────┘


   Error 

  Typed property AkrilliA\LaravelBeyond\NameResolver::$directory must not be accessed before initialization

  at vendor/akrillia/laravel-beyond/src/NameResolver.php:90
     86▕         $this->namespace = sprintf(
     87▕             $this->command->getNamespaceTemplate().'%s',
     88▕             $this->appOrDomain,
     89▕             $this->command->getType()->getNamespace(),
  ➜  90▕             $this->directory ? '\\'.$this->directory : '',
     91▕         );
     92▕ 
     93▕         $this->path = sprintf(
     94▕             '%s/'.$this->command->getFileNameTemplate(),

      +15 vendor frames 

  16  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
``` 

When u make `php artisan beyond:make:model Users/User` may not work, when u try `php artisan beyond:make:model Users.User` it works!

**Comment by alexanderkroneis** [comment]
We‘ll investigate asap, thanks for reporting!

**Comment by regnerisch** [comment]
Fixed in v7.0.0-beta.8
